### ISSUE TITLE
Code Javascript

### ISSUE THREAD

**Issue by Luanar01** [felipe-augusto/clean-code-javascript#22]

**Comment by hernandemonteiro** [comment]
You have a real problem?

**Comment by Luanar01** [comment]
No, i'm just studying. Seeking to find the best way to work with github in the future. 

- [x] ``

**Comment by hernandemonteiro** [comment]
Awesome, congratulations!

this place is better to get tech content and to study code with open sources projects.
The documentation is complete and you can explore more functionalitys.
